docker/fusionauth/.env

```diff
@@ -2,5 +2,6 @@ POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
 DATABASE_USERNAME=fusionauth
 DATABASE_PASSWORD=hkaLBM3RVnyYeYeqE3WI1w2e4Avpy0Wd5O3s3
-ES_JAVA_OPTS="-Xms512m -Xmx512m"
+OPENSEARCH_JAVA_OPTS="-Xms512m -Xmx512m"
```

Comment:
Can we alphabetize this?

Reply:
@Aaron-Ritter can you alphabetize these, please?